libs/sdk-ui-dashboard/src/model/commandHandlers/render/tests/__snapshots__/renderingWorker.test.ts.snap

```diff
@@ -0,0 +1,210 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renderingWorker async rendering resolution should accept new async rendering request, when it's fired within time limit after resolution 1`] = `
+Array [
+  Object {
+    "correlationId": undefined,
+    "type": "GDC.DASH/EVT.RENDER.REQUESTED",
+  },
+  Object {
+    "correlationId": undefined,
+    "type": "GDC.DASH/EVT.FILTER_CONTEXT.DATE_FILTER.VALIDATION.FAILED",
```

Comment:
This is suspicious a bit, why does the validation fail?

Reply:
Good point, I'll take a look.